src/CentreonRemote/Tests/Infrastructure/Service/ExportServiceTest.php

```diff
@@ -217,11 +215,11 @@ public function testImport()
 
         $this->fs->get('/export/')->add('manifest.json', new File($manifest));
 
-        $commitment = new ExportCommitment(null, null, null, null, $path, [
+        $this->export->import(new ExportCommitment(null, null, null, null, $path, [
             ConfigurationExporter::class,
-        ]);
+        ]));
 
-        $this->export->import($commitment);
+        $this->assertTrue(true);
```

Comment:
to prevent the showing of the warning

> 1) CentreonRemote\Tests\Infrastructure\Service\ExportServiceTest::testImport
This test did not perform any assertions
> 
> /home/vhr/Documents/StangaOne/projects/centreon/centreon-web/src/CentreonRemote/Tests/Infrastructure/Service/ExportServiceTest.php:199

Reply:
what do you want test?

Reply:
the test checks if the import method will throw an exception

Reply:
Can you use expectException?
https://stackoverflow.com/questions/5683592/phpunit-assert-that-an-exception-was-thrown

Reply:
for this case is not suitable, btw annotation is more convenient, here is you could see the implementation https://github.com/centreon/centreon/blob/abf59759e0311690bf058f24240ed9cfe6a5b99b/src/Centreon/Tests/Infrastructure/CentreonLegacyDB/CentreonDBAdapterTest.php#L139